src/js/badge.js

```diff
@@ -0,0 +1,48 @@
+import {isBadgeNonexisted} from './common/domUtil';
+import $ from 'jquery';
+
+const selectors = [
+  'h1.public > strong > a',                             //Specific repository
+  '.js-repo-filter .d-inline-block h3 a:first-child',   //Repositories & Stars
+  '.org-repos .d-inline-block a',                       //Organization
+  '.codesearch-results .d-inline-block a',              //Search
+  '.explore-pjax-container .explore-content a'          //Trending
+];
+const repoListSelectors = '.js-pinned-repos-reorder-container p.mb-0'; //Overview
+
+const showBadge = () => {
+  selectors.forEach((sel) => {
+    $(sel).each((i, element) => {
+      insertStatusBadge(element, element.pathname);
+    });
```

Comment:
`i` here not used. is the `each` function provided by jQuery? can it be done like `$(sel).forEach(element => insertStatusBadge(element, element.path))` with native JavaScript method? or the `each` method depends on the jQuery context or something?